### PR TITLE
Added 8.3 Companions

### PR DESCRIPTION
**Pull request by criggs11**

### CODE REVIEW

File: app/data/pets.json [hunk 1]
```diff
@@ -1519,6 +1596,115 @@
...
+            "creatureId": 161954,
+            "icon": "inv_babytentacleface",
+            "itemId": 174473,
+            "name": "K'uddly",
+            "spellid": 315339
+          },
+          {
+            "creatureId": 161992,
+            "icon": "ability_deathwing_grasping_tendrils",
+            "itemId": 173726,
```
Comment (seirl) [inline app/data/pets.json]: Wrong item ID here

File: app/data/pets.json [hunk 2]
```diff
@@ -454,6 +468,34 @@
...
+          },
+          {
+            "creatureId": 162006,
+            "icon": "ability_mount_camel_gray",
+            "itemId": 174477,
+            "name": "Pygmy Camel",
+            "spellid": 315363
+          },
+          {
+            "creatureId": 162007,
```
[the remark this answers is not in the document]
Comment (criggs11) [reply app/data/pets.json]: Fixed.

File: app/data/pets.json [hunk 3]
```diff
@@ -454,6 +468,34 @@
...
+          },
+          {
+            "creatureId": 162014,
+            "icon": "inv_icon_feather03a",
+            "itemId": 174480,
+            "name": "Windfeather Quill",
+            "spellid": 315372
+          },
+          {
+            "creatureId": 162013,
```
[the remark this answers is not in the document]
Comment (criggs11) [reply app/data/pets.json]: Fixed.

File: app/data/pets.json (hunk 1 above)
Comment (criggs11) [reply app/data/pets.json]: Fixed.

**Comment by seirl** [comment]
Thanks, have you double-checked the other IDs too? I haven't really had time to review every single ID.

**Comment by criggs11** [comment]
> Thanks, have you double-checked the other IDs too? I haven't really had time to review every single ID.

Let me triple check to be sure.

**Comment by criggs11** [comment]
> > Thanks, have you double-checked the other IDs too? I haven't really had time to review every single ID.
> 
> Let me triple check to be sure.

✅ Triple checked the companion item id's.

**Comment by seirl** [comment]
What about the spell/npc ids ? :-)

In the future you can avoid work by using the scripts in data_scripts/ that automatically fill the blanks for you. Now that it's already done I'm not sure there's something that could triple check out of the box, but it would probably be nice to have. Actually it could even be integrated in the test suite.

**Comment by criggs11** [comment]
> What about the spell/npc ids ? :-)
> 
> In the future you can avoid work by using the scripts in data_scripts/ that automatically fill the blanks for you. Now that it's already done I'm not sure there's something that could triple check out of the box, but it would probably be nice to have. Actually it could even be integrated in the test suite.

Yep, I checked the npc ids and spell ids in addition to the item ids. I didn't realize there was a script, I just did this all manually. Would be nice to have a validation test though.

**Comment by seirl** [comment]
Thank you for your work!

**Pull request merged**